rfcs/20220218-level_zero-headers/README.md

```diff
@@ -0,0 +1,93 @@
+# Adding Level Zero header to oneDNN sources
+
+This RFC discusses the option of adding the Level Zero headers to the
+oneDNN repository.
+
+On Windows, Level Zero headers are not distributed as part of oneAPI
```

Comment:
Could you please explain what is the reason for this limitations of Level Zero distribution on Windows?

Reply:
FWIW, my guess is that driver on Windows contains only runtime components, while headers are devel ones. But this is strange anyway.